aptos-move/move-examples/parallel_accounts/Move.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "ParallelAccountsDemo"
+version = "1.0.0"
+
+[dependencies]
+AptosFramework = { local = "../../framework/aptos-framework" }
```

Comment:
This is not technically needed I think since AptosToken already pulls in the framework dependency

Reply:
Furthermore due to lack of dynamic dispatch here, you can't perform arbitrary operations against this account. Users can perhaps use the script to obtain the signer then run operations, but scripts are currently not that easy to use.

Reply:
This is purposely written as code instead of scripts for readability.  Ideally, it would be a script instead, but portability and everything that comes with it isn't ideal.